entrypoint

```diff
@@ -14,5 +14,10 @@ if [[ "$rippledconfig" -gt "0" && "$validatorstxt" -gt "0" ]]; then
 
 fi
 
+ENV_ARGS = ""
+if [[ ! -z "${STANDALONE_MODE}" ]]; then
+  ENV_ARGS="${ENV_ARGS} -a --start"
+fi
+
 # Start rippled, Passthrough other arguments
-exec /opt/ripple/bin/rippled --conf /etc/opt/ripple/rippled.cfg "$@"
+exec /opt/ripple/bin/rippled --conf /etc/opt/ripple/rippled.cfg $ENV_ARGS "$@"
```

Comment:
@WietseWind I'm not quite sure how to use the new version to get rippled to start in standalone mode. It seems that you can pass in environment variables with `-e`, but the environment variables aren't used in the actual startup script. 

Maybe this is the missing line? (On master, the entrypoint still only has the `"$@"` and not the `$ENV_ARGS` parameter)

Ideally I'd like something like the following script to start a rippled standalone mode: 
```
docker run -dit \
    --name rippled \
    -p 80:80 \
    -v /my/local/disk/xrpld-config/:/config/ \
    -e ENV_ARGS="-a --start" \
    xrpllabsofficial/xrpld:latest
```

Reply:
The problem is I can't move anything after the image name in GitHub actions setup, which is why I was hoping to work around it with a special environment variable like `ENV_ARGS` which could be passed in as if they were in the normal arg position. 

Having rippled as a `service` let's you pass in options like `-e`, but doesn't have support for actual args. 

https://github.com/XRPLF/xrpl.js/blob/0fa5415d297845c6f59d7c0e350cfb6dd7d199ea/.github/workflows/nodejs.yml#L103-L108

Reply:
Clear, I see. That makes sense. Ok, I'll publish an update with ENV_ARGS then, which will be appended after the native arguments.